docs/base/forms.md

```diff
@@ -55,6 +62,13 @@ Use radio buttons to select one or more options, our radio buttons can appear in
 View example of the base radio buttons
 </a>
 
+By default, checkboxes are vertically aligned to the baseline of text wrapped in a `label`, `h5`, `h6`, or `p` tag. If you need to align it to other elements, use the classes below:
```

Comment:
The classes aren't visible unless the use inspects the markup. Can you change either the text here or the visible text in the example to show the classes, please?